utils/ci/corefile_in_gha.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd daos
+
+echo ::group::Check for corefiles
+if [ $BASE_DISTRO == "ubuntu" ]
+then
+    # "apport" corefile repo
+    COREFILE_DIR="/var/crash/"
+else
+    COREFILE_DIR="/var/lib/systemd/coredump/"
+fi
+for i in `ls -1 $COREFILE_DIR/` ; do
```

Comment:
(lint) Iterating over ls output is fragile. Use globs. [SC2045]
(lint) Use $(...) notation instead of legacy backticks `...`. [SC2006]